jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+		"moduleResolution": "node",
+		"compilerOptions": {
+				"jsx": "preserve",
+				"baseUrl": ".",
+				"paths": {
+						"@/*": ["*"]
+				}
+		},
+		"exclude": ["node_modules", ".next"]
+}
```

Comment:
do we need jsconfig? is this the file that allows to use import path aliases with @?

Reply:
Yes it is, and I have just resolved the @ alias error messages you had. Kindly check fresh pull for confirmation.

Reply:
yes, its working now @CharaD7 , I think this PR can be merged now

Reply:
actually hold on, I see the auto formatting of code as per our prettier config on save file is not working in this brach.

Reply:
looks like 'plugin:jest/recommended' is overriding our prettier rules, removing it from bslintrc's extensions brings back the prettier formatting. 
Do we need this? I think we can remove 'plugin:jest/recommended'